src/Components/ListofTodo/ListofTodo.tsx

```diff
@@ -0,0 +1,23 @@
+import { Todo } from '../../types/Todo';
+import { Todos } from '../Todos/Todos';
+
+type Props = {
+  todos: Todo[];
+  onDeleteTodo: (id: number) => void;
+  temporaryTodo: Todo | undefined;
+};
+
+export const ListofTodo: React.FC<Props> = ({
+  todos,
+  onDeleteTodo,
+  temporaryTodo,
+}) => (
+  <>
+    {todos.map((todo) => (
+      <Todos todo={todo} key={todo.id} onDelete={onDeleteTodo} />
+    ))}
+    {temporaryTodo && (
+      <Todos todo={temporaryTodo} key={temporaryTodo.id} onDelete={() => {}} />
```

Comment:
Instead of passing an empty function for the `onDelete` prop in the `Todos` component, you can remove it from props and check inside the component if this function exist